17_1_shape.js

```diff
@@ -0,0 +1,64 @@
+const ctx = document.querySelector('canvas').getContext('2d');
+// A trapezoid
+
+ctx.strokeStyle = 'black';
```

Comment:
Maybe collect all these expressions into functions like `generateTrapezoid` etc.

Reply:
yup, if you'll try to open that file you'll see that variable declarations for `x` are duplicated - but them into separate scopes